nemo/collections/tts/models/talknet.py

```diff
@@ -134,20 +134,23 @@ class TalkNetPitchModel(ModelPT):
 
     def __init__(self, cfg: DictConfig, trainer: 'Trainer' = None):
         super().__init__(cfg=cfg, trainer=trainer)
+
+        print(cfg)
+
         typecheck.set_typecheck_enabled(enabled=False)
```

Comment:
Remove all `typecheck.set_typecheck_enabled(enabled=False)` from your model.

Reply:
If I do remove enabled=False line, then I require to type all forward methods.
Just to clarify: is this a requirement now again?

Reply:
It always was. I must have missed it when I first approved TalkNet

Reply:
fixed